docker/Dockerfile

```diff
@@ -58,28 +36,37 @@ RUN conda update -qy conda \
       pip \
       python=3.7
 
-COPY . /app/alphafold
-RUN wget -q -P /app/alphafold/alphafold/common/ \
-  https://git.scicore.unibas.ch/schwede/openstructure/-/raw/7102c63615b64735c4941278d92b554ec94415f8/modules/mol/alg/src/stereo_chemical_props.txt
+COPY . /home/jupyter/alphafold
 
 # Install pip packages.
 RUN pip3 install --upgrade pip \
-    && pip3 install -r /app/alphafold/requirements.txt \
-    && pip3 install --upgrade jax jaxlib==0.1.69+cuda${CUDA/./} -f \
+    && pip3 install -r /home/jupyter/alphafold/requirements.txt \
+    && pip3 install --upgrade jax jaxlib==0.1.69+cuda110 -f \
       https://storage.googleapis.com/jax-releases/jax_releases.html
+   
+# Downgrade because of new base image.
+RUN pip3 install jax==0.2.14 py3Dmol tqdm
+
+# Install alphafold.
+WORKDIR /home/jupyter/alphafold
+RUN python setup.py install
 
 # Apply OpenMM patch.
 WORKDIR /opt/conda/lib/python3.7/site-packages
-RUN patch -p0 < /app/alphafold/docker/openmm.patch
+RUN patch -p0 < /home/jupyter/alphafold/docker/openmm.patch
+
+# Creating a tmp location for jackhmmr; not mounting through to host though.
+RUN sudo mkdir -m 777 --parents /tmp/ramdisk
```

Comment:
mkdir, but not mounting; jackhmmer will use /tmp/ramdisk, but it is not mounted through to the host